src/common/AndroidUtils.ts

```diff
@@ -777,12 +501,12 @@ export class AndroidUtils {
      * @param numRetries Optional parameter that indicates the number of times the command will be executed again if ADB comes back with an error. Defaults to AndroidConfig.AdbNumRetryAttempts.
      * @param retryDelay Optional parameter that indicates the milliseconds of delay before retrying again. Defaults to AndroidConfig.AdbRetryAttemptDelay.
      */
-    public static async executeAdbCommandWithRetry(
+    public static async executeAdbCommand(
```

Comment:
I'm making a change to run ALL `adb` commands with retry mechanism. I don't know why we didn't do this in the first place along time ago. It's just a good thing to do since on Windows `adb` barf a lot and we need to run a command again for it to pass.

So by default an `adb` command now will be retried. If the caller really doesn't want the retry mechanism then they can pass in 0 for `numRetries`